<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRelink">
        <property name="text">
         <string>Relink</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="lePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textArea">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Recent">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
     <addaction name="actionRecent1"/>
     <addaction name="actionRecent2"/>
     <addaction name="actionRecent3"/>
     <addaction name="actionRecent4"/>
     <addaction name="actionRecent5"/>
     <addaction name="actionRecent6"/>
    </widget>
    <addaction name="actionRelink"/>
    <addaction name="menu_Recent"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionClear_Output"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRelink">
   <property name="text">
    <string>&amp;Relink</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRecent1">
   <property name="text">
    <string>recent1</string>
   </property>
  </action>
  <action name="actionRecent2">
   <property name="text">
    <string>recent2</string>
   </property>
  </action>
  <action name="actionRecent3">
   <property name="text">
    <string>recent3</string>
   </property>
  </action>
  <action name="actionRecent4">
   <property name="text">
    <string>recent4</string>
   </property>
  </action>
  <action name="actionRecent5">
   <property name="text">
    <string>recent5</string>
   </property>
  </action>
  <action name="actionRecent5_2">
   <property name="text">
    <string>recent5</string>
   </property>
  </action>
  <action name="actionRecent6">
   <property name="text">
    <string>recent6</string>
   </property>
  </action>
  <action name="actionClear_Output">
   <property name="text">
    <string>Clear Output</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
